File: Algorithm/C++/1000+/1550. Three Consecutive Odds.cc
#include <vector>

class Solution {
 public:
    bool threeConsecutiveOdds(std::vector<int>& arr) {
       int cnt=0;
       for(auto num: arr) {
           if (num&1 > 0) {
               cnt++;
               if (cnt==3) return true;
           } else {
               cnt = 0;
           }
       }
       return false;
    }
};

File: Algorithm/Python/1000+/1550. Three Consecutive Odds.py
class Solution(object):
    def threeConsecutiveOdds(self, arr):
        """
        :type arr: List[int]
        :rtype: bool
        """
        cnt = 0
        for num in arr:
            if num&1:
                cnt += 1
                if cnt == 3:
                    return True
            else:
                cnt = 0
        return False